h2o-py/tests/testdir_misc/pyunit_columns_by_type.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import print_function
+from builtins import range
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+
+def lists_equal(l1,l2):
+    return len(l1) == len(l2) and sorted(l1) == sorted(l2)
+
+def pyunit_columns_by_types():
+
+    fr = h2o.import_file(pyunit_utils.locate("smalldata/jira/filter_type.csv"))
```

Comment:
This will cause all jobs to fail until Jenkins (and all users) re-sync their smalldata... 
A better approach would be to create the dataset right here from a python object (it's not that big anyway).